all/pom.xml

```diff
@@ -83,8 +89,31 @@
     </build>
 
     <profiles>
+        <!-- Default package is built for AEM -->
+        <profile>
+            <id>aem</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
check if build definition needs update because when passing a profile the active by default profiles won't be active ;)